readthedocs/templates/profiles/private/token_list.html

```diff
@@ -0,0 +1,34 @@
+{% extends "profiles/base_profile_edit.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Personal Access Tokens" %}{% endblock %}
+
+{% block profile-admin-tokens %}active{% endblock %}
+
+{% block edit_content_header %} {% trans "Personal Access Tokens" %} {% endblock %}
+
+{% block edit_content %}
```

Comment:
We definitely need something saying that users can't set them up yet, and to contact us to create one.

Reply:
I added a message. Let me know.

![Screenshot_2019-07-22_13-55-38](https://user-images.githubusercontent.com/244656/61630606-6dd95480-ac88-11e9-8c3b-d26e21706cd3.png)